Xamarin.Forms.Core/Expander.cs

```diff
@@ -182,6 +184,13 @@ protected override void OnBindingContextChanged()
 			SetContent(true, true);
 		}
 
+		protected override void OnPropertyChanged([CallerMemberName] string propertyName = null)
```

Comment:
@rmarinho do you think it would be better to subscribe PropertyChanged event in the constructor instead of overriding this method?

Reply:
No better do it here imo, so the framework will take care of unsubscribe if needed.

Reply:
> No better do it here imo, so the framework will take care of unsubscribe if needed.

I am worried that somebody can override OnPropertyChanged and do not call the base method.

Reply:
Thats there fault :p